data/json/vehicles/vans_busses.json

```diff
@@ -1271,6 +1271,89 @@
+      { "x": 2, "y": 1, "parts": [ "frame_horizontal", "halfboard_horizontal" ] },
+      { "x": 2, "y": 1, "part": "tank", "fuel": "gasoline" },
+      { "x": 2, "y": -1, "parts": [ "frame_nw", "halfboard_nw", "headlight", "wheel_steerable" ] },
+      { "x": 2, "y": 2, "parts": [ "frame_ne", "halfboard_ne", "headlight", "wheel_steerable" ] },
+      { "x": -1, "y": 0, "parts": [ "frame_horizontal", "hatch_opaque" ] },
+      { "x": -1, "y": 1, "parts": [ "frame_horizontal", "hatch_opaque" ] },
+      { "x": -1, "y": -1, "parts": [ "frame_vertical", "halfboard_vertical" ] },
+      { "x": -1, "y": 2, "parts": [ "frame_vertical", "halfboard_vertical" ] },
+      { "x": -2, "y": 0, "parts": [ "frame_vertical_2", "minifridge", "roof" ] },
+      { "x": -2, "y": 1, "parts": [ "frame_vertical_2", "seat", "seatbelt", "roof" ] },
+      { "x": -2, "y": -1, "parts": [ "frame_vertical", "halfboard_vertical" ] },
+      { "x": -2, "y": 2, "parts": [ "frame_vertical", "halfboard_vertical" ] },
+      { "x": -3, "y": 0, "parts": [ "frame_vertical_2", "aisle_lights", "veh_table", "roof" ] },
+      { "x": -3, "y": 1, "parts": [ "frame_vertical_2", "seat", "seatbelt", "roof" ] },
+      { "x": -3, "y": -1, "parts": [ "frame_vertical", "halfboard_vertical" ] },
+      { "x": -3, "y": 2, "parts": [ "frame_vertical", "halfboard_vertical" ] },
+      { "x": -4, "y": 0, "parts": [ "frame_vertical_2", "aisle_vertical", "aisle_lights", "roof" ] },
+      { "x": -4, "y": 1, "parts": [ "frame_vertical_2", "seat", "seatbelt", "roof" ] },
+      { "x": -4, "y": -1, "parts": [ "frame_vertical", "halfboard_vertical" ] },
+      { "x": -4, "y": 2, "parts": [ "frame_vertical", "halfboard_vertical" ] },
+      { "x": -5, "y": 0, "parts": [ "frame_vertical_2", "aisle_vertical", "roof" ] },
+      { "x": -5, "y": 1, "parts": [ "frame_vertical_2", "aisle_vertical", "roof" ] },
+      { "x": -5, "y": -1, "parts": [ "frame_vertical", "door" ] },
+      { "x": -5, "y": 2, "parts": [ "frame_vertical", "door" ] },
+      { "x": -6, "y": 0, "parts": [ "frame_vertical_2", "seat", "seatbelt", "roof" ] },
+      {
+        "x": -6,
+        "y": 1,
+        "parts": [ "frame_vertical_2", "battery_car", "aisle_lights", "box", "recharge_station", "controls_electronic" ]
+      },
+      { "x": -6, "y": -1, "parts": [ "frame_vertical", "halfboard_vertical" ] },
+      { "x": -6, "y": 2, "parts": [ "frame_vertical", "halfboard_vertical" ] },
+      { "x": -7, "y": 0, "parts": [ "frame_horizontal", "halfboard_horizontal" ] },
+      { "x": -7, "y": 1, "parts": [ "frame_horizontal", "halfboard_horizontal" ] },
+      { "x": -7, "y": -1, "parts": [ "frame_sw", "halfboard_sw", "wheel", "roof" ] },
+      { "x": -7, "y": 2, "parts": [ "frame_se", "halfboard_se", "wheel", "roof" ] }
+    ],
+    "items": [
+      { "x": -6, "y": 1, "chance": 80, "items": "light_plus_battery_cell" },
+      { "x": -6, "y": 1, "chance": 80, "items": "light_plus_battery_cell" },
```

Comment:
Create some `item_group` with the batteries you wanted, and add a `"charges-min": XXX` and `"charges-max": YYY` fields like this:

```
{
    "type": "item_group",
    "id": "limousine_batteries",
    "items": [ { "item": "light_plus_battery_cell", "prob": 80, "charges-min": XXX, "charges-max": YYY } ]
}
```

And then make this item group spawn instead of `items`.

Reply:
Thanks, I don't know why but I tried to search for one item group with light bats and didn't find them. I searched again and found "{ "x": -6, "y": 1, "chance": 80, "item_groups": "ammo_light_batteries_full" in `item_groups/ammo.json`. I guess I didn't do a recursive directory search last time.

I wanted specifically recharged bats there, but I guess that group will work fine too (also some throws in some random drop results)